packages/synapse-react-client/src/synapse-queries/user/useGetUserTeams.ts

```diff
@@ -7,24 +7,28 @@ import {
 import SynapseClient from '../../synapse-client'
 import { SynapseClientError } from '../../utils/SynapseClientError'
 import { useSynapseContext } from '../../utils/context/SynapseContext'
-import { PaginatedIds, PaginatedResults } from '@sage-bionetworks/synapse-types'
-import { Team } from '@sage-bionetworks/synapse-types'
+import {
+  PaginatedIds,
+  PaginatedResults,
+  Team,
+} from '@sage-bionetworks/synapse-types'
 import { getNextPageParamForPaginatedResults } from '../InfiniteQueryUtils'
 
-export function useGetUserSubmissionTeamsInfinite(
+export function useGetUserSubmissionTeams(
   challengeId: string,
   limit?: number,
+  offset: number = 10,
```

Comment:
Why would the offset default to 10 instead of 0?  Did you intend to set the limit default to 10?

Reply:
Yes, good catch